client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -64,17 +70,21 @@ class CancelPurchaseForm extends Component {
 		cancelBundledDomain: PropTypes.bool,
 		includedDomainPurchase: PropTypes.object,
 		linkedPurchases: PropTypes.array,
+		skipSurvey: PropTypes.bool,
 	};
 
 	static defaultProps = {
 		isVisible: false,
 	};
 
 	getAllSurveySteps() {
-		const { willAtomicSiteRevert, purchase } = this.props;
+		const { purchase, skipSurvey, willAtomicSiteRevert } = this.props;
 		let steps = [ FEEDBACK_STEP ];
 
-		if ( isPartnerPurchase( purchase ) && isAgencyPartnerType( purchase.partnerType ) ) {
+		if (
+			skipSurvey ||
+			( isPartnerPurchase( purchase ) && isAgencyPartnerType( purchase.partnerType ) )
+		) {
 			/**
 			 * We don't want to display the cancellation survey for sites purchased
 			 * through partners (e.g., A4A.)
```

Comment:
This comment is outdated. `skipSurvey` isn't limited to sites purchased through partners

Reply:
@rcrdortiz good catch, it's now removed.